docs/development/docrules.rst

```diff
@@ -61,7 +61,8 @@ function, class, or method definition. The docstring is a special attribute
 of the object (``object.__doc__``) and, for consistency, is surrounded by
 triple double quotes, i.e.::
 
-   """This is the form of a docstring.
+   """
+   This is the form of a docstring.
```

Comment:
Could you explain why you changed this line?

I always thought PEP8 explicitly supported this kind of docstring, see for example all Examples in [PEP-257 (Docstring conventions)](https://www.python.org/dev/peps/pep-0257/) and [PEP-8 Documentation strings summary](https://www.python.org/dev/peps/pep-0008/#documentation-strings). Actually these also state that one line docstrings should also have the trailing `"""` in the same line.